### ISSUE TITLE
CJK Sans?

### ISSUE THREAD

**Issue by MAZ06** [Mercury13/unicodia#353]
Why are CJK characters in a sans font? Shouldn't they be serif like (almost) everything else?

**Comment by Mercury13** [comment]
I mostly prefer serif, but depends on “the most canonical” look of letters for foreigners.
Serif: Han ideographs regardless of country, old CJK scripts.
Sans: Kana (except Hentaigana, I’ve found a very interesting font where Hentaigana is sans → scrapped), Hangul

Tamil just uses standard Windows font :)
About Telugu, Malayalam, Myanmar — I even don’t know.
Javanese — at least W10 has good Javanese, but because of long ascenders/descenders I could not fit it to the cell :)
Lanna — recently found
Odia, Gurmukhi, Gujarati — I’m glad that Noto made a cool semi-serif font
Ethiopic — I prefer normal serif to “canonical” calligraphic font, and I think the best version for Ethiopic is semi-serif like on Mac
Same for Caucasian Albanian: there is a grave of scholar who studied it, and it was written in nice semi-serif — just want! (UPD: Caucasian is sans, semi-serif is Georgian, but still want such Caucasian)
Nüshu — want some sort of serif
Old scripts scratched on something like Brahmi or Parthian — just sans
Devanagari, Bengali, Kannada — only serif!
Khojki — does not really matter, but Noto serif is so nice!
Old Indic — probably sans because of method of writing (on copper foil or palm leaves)
Gothic is so “gothic” that I specifically looked for serif!
Makasar — want special “bird” serif as written on treaties with Dutch East-Indic company

**Comment by Mercury13** [comment]
About new scripts like Toto — it’s better to base fonts on handwritten specimen. Specifically Toto — cool people from SIL made a canonical font, but IDK what to take for Tangsa, canonical letters are being drawn right away. SIL also made a font for Kirat, also based on handwriting rather than on Unicode images.

**Comment by Mercury13** [comment]
Javanese is Noto, just because of those funky metrics! And because of W7, of course: just a few days ago I proclaimed that I have nothing for W7 except simple runnability, the former policy was the entire base plane and three P1 scripts of extreme importance.

User fonts glitch sometimes, and those glitches are hard to repeat and fix.

Someday I will move Korean to serif, but kana will remain sans. It’ll also improve Han radicals and Bopomofo.

Which scripts are done really well with Noto: Canadian, probably Tangsa (IDK), Mongol (except glyphs).

I prefer other fonts if possible. For Arabic I use SIL’s Scheherazade.

**Comment by MAZ06** [comment]
What's wrong with Noto Arabic?

**Comment by Mercury13** [comment]
IDK about complex Arabic rendering, but it adopted new characters very late.

**Comment by MAZ06** [comment]
Should be okay to switch to Noto Arabic now, right?

I'm looking forward to #314.

**Comment by Mercury13** [comment]
Noto Arabic — IDK.
We’ll see in the autumn, who will draw four new characters first. In U14 and 15 Scheherazade was long ahead.

**Comment by Mercury13** [comment]
@MAZ06 
I want to take a sabbatical and stop doing something big.
But at least I’ll draw a mockup of how I see #314 Arabic.
![image](https://github.com/Mercury13/unicodia/assets/10164882/12505932-ff8c-41a3-83ae-2b73d10963ee)

**Comment by MAZ06** [comment]
![ken-jeong-community](https://github.com/Mercury13/unicodia/assets/165231781/a8d22092-db67-4e61-ba52-5adcf4c117db)

**Comment by MAZ06** [comment]
The forms are also not labeled... :(

**Comment by MAZ06** [comment]
Wait... Those aren't even the actual forms... lol

**Comment by Mercury13** [comment]
@MAZ06 I don’t want to make styles like in other two channels, Glagolitic or vertical.
Yes, that’s a mockup.
And here’s the old thing — Glagolitic.
![image](https://github.com/Mercury13/unicodia/assets/10164882/73ade4a6-a619-4cb7-a2aa-77506282bdbf)

**Comment by Mercury13** [comment]
You are quite right, but that removes one thing — selection what style to draw in table. Does it matter?

**Comment by Mercury13** [comment]
@MAZ06 
I want to build a mixed sans-serif font, so Hangul and Bopomofo will be serif. Hard, probably.
